frontend/src/features/projects/list/columns.tsx

```diff
@@ -34,15 +39,20 @@ export const columns = (onDelete?: (id: string) => void): ColumnWithProps<IProje
       Cell: (props: CellProps<IProject>) => {
         return (
           <div>
-            {!!onDelete && (
-              <FaTrash
-                style={{ marginRight: 10, cursor: 'pointer' }}
-                onClick={(e: any) => {
-                  e.stopPropagation();
-                  onDelete(props.row.original.projectNumber);
-                }}
-              />
-            )}
+            {/* delete button is visible when on delete button function is passed, if it is in draft and they have the edit claim, they are an admin, or it is their project before submission */}
+            {!!onDelete &&
+              ((projectEditClaim && props.row.original.projectNumber.includes('DRAFT')) ||
+                isAdmin ||
+                (user === props.row.original.updatedBy &&
+                  props.row.original.projectNumber.includes('DRAFT'))) && (
+                <FaTrash
+                  style={{ marginRight: 10, cursor: 'pointer' }}
+                  onClick={(e: any) => {
+                    e.stopPropagation();
```

Comment:
does this stop the onClick for the row or something as well?

Reply:
Allows the delete button to be clicked separately from the row onClick!